src/SonarScanner.MSBuild.Tasks/Targets/SonarQube.Integration.targets

```diff
@@ -128,6 +128,7 @@
   <UsingTask TaskName="GetAnalyzerSettings" AssemblyFile="$([MSBUILD]::Unescape($(SonarQubeBuildTasksAssemblyFile)))" />
   <UsingTask TaskName="MakeUniqueDir" AssemblyFile="$([MSBUILD]::Unescape($(SonarQubeBuildTasksAssemblyFile)))" />
   <UsingTask TaskName="WriteZeroLengthFiles" AssemblyFile="$([MSBUILD]::Unescape($(SonarQubeBuildTasksAssemblyFile)))" />
+  <UsingTask TaskName="WriteTelemetry" AssemblyFile="$([MSBUILD]::Unescape($(SonarQubeBuildTasksAssemblyFile)))" />
```

Comment:
I just realized that the TaskName can [not be changed in this import](https://learn.microsoft.com/en-us/visualstudio/msbuild/usingtask-element-msbuild?view=vs-2022). `WriteTelemetry` is quite generic and because we run during build with many potential other parties adding their own Tasks, there is a risk of a naming collision. We should rename the Task to something like `WriteSonarTelemetry` instead.
I created https://sonarsource.atlassian.net/browse/SCAN4NET-492